ontology/cdk.owl

```diff
@@ -42,7 +42,7 @@
         <dc:format rdf:datatype="&xsd;string">application/rdf+xml</dc:format>
         <dc:title rdf:datatype="&xsd;string">CDK descriptor algorithm implementations</dc:title>
         <dc:language rdf:datatype="&xsd;string">en</dc:language>
-        <dc:rights>http://creativecommons.org/licenses/by/3.0</dc:rights>
+        <dc:rights>https://creativecommons.org/publicdomain/zero/1.0/legalcode</dc:rights>

```

Comment:
```suggestion
         <dc:rights>https://creativecommons.org/publicdomain/zero/1.0</dc:rights>
```

Reply:
Note that all of these should probably be `dc:license` based on #31

Reply:
Fixed.